README.md

```diff
@@ -93,8 +93,11 @@ It is based upon and contains a lot of code written by [Jeroen De Dauw]
 
 ## Release notes
 
-### 2.0.0 (dev)
+### 2.0.0 (2017-05-08)
 
+* `GlobeCoordinateValue` does not accept empty strings as globes any more.
+* `GlobeCoordinateValue` does not accept precisions outside the [-360..+360] interval any more.
+* Changed hash calculation of `GlobeCoordinateValue` to be more robust.
```

Comment:
Shouldn't this explicitly say the hash change was a backwards incompatible one? I know "robust" is supposed to mean it here but if does not immediately strike as a breaking change IMO.